src/introduction.md

```diff
@@ -40,6 +40,16 @@ relm4-components = "0.1"
 ```
 
 # Examples
+In order to get you quickly started, we have prepped some better examples to explain the Rem4 features.
```

Comment:
To get you started quickly, Relm4 provides a lot of examples that you can use to learn about different features or as an inspiration for your application.